### PR TITLE
New training docs to activities

### PR DESCRIPTION
**Pull request by AnthonyRollins**

### CODE REVIEW

File: docs/contacts/activity/create-note.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# How to create a note for a contact record
+
+!!! Note
+    Below showcases how to add a note to a contact record, but this workflow can be used to also add a note to company.
```
Comment (laneymangan) [inline docs/contacts/activity/create-note.md:4]:
```suggestion
    Below showcases how to add a note to a contact record, but this workflow can be used to also add a note to a company record.
```

File: docs/contacts/activity/create-task.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+# Create a Task
+
+!!! Note
+    There are several ways to create a task, including from within the tasks index page, an object index page, an individual record, or by automating through a deal phase. The following will demonstrate how to create a task starting from within the task index page.**
```
Comment (laneymangan) [inline docs/contacts/activity/create-task.md:4]:
```suggestion
    There are several ways to create a task, including from within the tasks index page, an object index page, an individual record, or by automating through a deal phase. The following will demonstrate how to create a task starting from within the task index page.
```

File: docs/contacts/activity/logging-calls.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
+# Hubspot: How to log a call
+
+!!! Note
+   The purpose of the CRM is to have a centralized location for all communication with agencies we work with. Logging calls is an important way to track communication.
```
Comment (laneymangan) [inline docs/contacts/activity/logging-calls.md:4]: The text here (The purpose of the CRM is to have a centralized location for all communication with agencies we work with. Logging calls is an important way to track communication.) is not inside the Note box. Not sure it needs to be but as it is now, there's an empty Note box

File: docs/contacts/activity/log-meeting.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
+# Hubspot: Log a meeting
```
Comment (laneymangan) [inline docs/contacts/activity/log-meeting.md:1]:
```suggestion
# Log a meeting
```

Comment (laneymangan) [reply docs/contacts/activity/log-meeting.md:1]: I would suggest either adding "Hubspot:" to all of the trainings or removing it from all just for consistency

File: docs/contacts/activity/logging-calls.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
+# Hubspot: How to log a call
```
Comment (laneymangan) [inline docs/contacts/activity/logging-calls.md:1]:
```suggestion
# How to log a call
```

Comment (laneymangan) [reply docs/contacts/activity/logging-calls.md:1]: I would suggest either adding "Hubspot:" to all of the trainings or removing it from all just for consistency

File: docs/contacts/activity/activity.md [hunk 6]
```diff
@@ -2,8 +2,8 @@
 
 Activities are interactions that a team member has had with a contact and are recorded on that record’s timeline. Activities include emails, calls, tasks, notes and meetings. Activities also have the ability to be filtered so that team members can view last communication, ticket replies, or a variety of other helpful updates.
 
-- [Training: How to Log an email](/docs/contacts/activity/logging-emails.md)
```
Comment (laneymangan) [inline docs/contacts/activity/activity.md:5]: The steps within the logging email trainings aren't numbered, not sure that's a huge deal but wanted to point that out

Comment (AnthonyRollins) [reply docs/contacts/activity/activity.md:5]: Yeah not a huge deal, but I think it should be fixed.

**Pull request merged**